packages/apps/crm/src/components/molecules/SocialNetworksInfoCard/SocialNetworksInfoCard.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {LabelText, checkNullString, useThemeColor} from '@axelor/aos-mobile-ui';
+import {SocialNetworkLinks} from '@axelor/aos-mobile-core';
+
+interface SocialNetworksInfoCardProps {
+  style?: any;
+  title: string;
+  headerIconName: string;
+  FontAwesome5HeaderIcon?: boolean;
+  border?: boolean;
+  styleBorder?: any;
+  name?: string;
+  lastName?: string;
+  company?: string;
+}
```

Comment:
Simplify here by leaving only props that can change (title & icon are fixed)

Why does `lastName` have default value but not the other props ? I think none should have a default value.